src/pages/QuotesEstimatesPage/QuotesEstimatesPage.js

```diff
@@ -1,25 +1,191 @@
+
 import React from 'react';
-import Header from '../../components/organisms/Header';
-import QuoteDetails from '../../components/molecules/QuoteDetails';
-import Button from '../../components/atoms/Button';
 import styled from 'styled-components';
 
 const PageWrapper = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
+  padding: 20px;
+  background-color: #fff;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  max-width: 800px;
+  margin: 0 auto;
+`;
+
+const Header = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  padding: 20px;
+  border-bottom: 1px solid #000;
+`;
+
+const HeaderTitle = styled.h1`
+  font-size: 24px;
+  font-weight: 700;
+  color: #000;
+`;
+
+const Sidebar = styled.div`
+  width: 200px;
+  background-color: #31373d;
+  display: flex;
+  flex-direction: column;
+  padding: 10px 0;
+`;
+
+const SidebarItem = styled.div`
+  color: #f1f2f2;
+  padding: 15px 20px;
+  font-size: 16px;
+  cursor: pointer;
+  &.logo {
+    background-color: #000;
+    color: #fff;
+    font-weight: 700;
+  }
+  &.active {
+    background-color: #497be8;
+  }
+  &:hover {
+    background-color: #3d4349;
+  }
+`;
+
+const MainContent = styled.div`
+  padding: 20px;
+  background-color: #fff;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  max-width: 800px;
+  margin: 0 auto;
+`;
+
+const TableHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const TableHeaderItem = styled.div`
+  flex: 1;
```

Comment:
The MainContent component has a fixed max-width. Ensure this does not affect the responsiveness on smaller screens.